dump25/len.py

```
@@ -0,0 +1,18 @@
+import os
+import bz2
```

Ruff (0.8.2)

`bz2` imported but unused

Remove unused import: `bz2`

(F401)